source/content/dns.md

```diff
@@ -162,39 +159,27 @@ Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (t
 
 Pantheon provides IPv6 addresses for you to assign `AAAA` records to. If your DNS service does not support `AAAA` you can simply omit these records for now, but consider asking your provider to add functionality, as the need for IPv6 will only increase in the future.
 
-Some DNS service providers require expanded AAAA records. You can retrieve the expanded version for each AAAA record by using the [IPv6 conversion tool](http://ipv6-literal.com/) and copying the "IPv6 longest" result.
-
+Some DNS service providers require expanded `AAAA` records. You can retrieve the expanded version for each AAAA record by using the [IPv6 conversion tool](http://ipv6-literal.com/) and copying the "IPv6 longest" result.
 
 ### What about my `MX` records for email?
 
-Pantheon does not provide email services. Make sure your DNS records include an `MX` reacord that points to a subdomain (like `mail`), which in turn has an `A` or `CNAME` record pointing it to your email provider.
+Pantheon does not provide email services. Make sure your DNS records include an `MX` record that points to a subdomain (like `mail`), which in turn has an `A` or `CNAME` record pointing it to your email provider.
 
 ### What is the difference between an `A` and `CNAME` record?
 
 An `A` record points a domain name to an IPv4 address. A `CNAME` record points a domain name to *another* domain, but *does not* redirect to it.
 
-Pantheon will provide both `A` and `CNAME` values, depending on the domains provided:
+Pantheon provides `A` and `AAAA` values:
 
 ![DNS Values provided by the Pantheon Site Dashboard](../images/dashboard/dns-values.png)
 
-If your DNS provider doesn't allow `CNAME` records, you can use `nslookup` to determine the IP address for the domain value, and then supply that as an `A` record:
-
-```bash
-$ nslookup live-anita-drupal.pantheonsite.io
-Server:         192.168.1.1
-Address:        192.168.1.1#53
-
-Non-authoritative answer:
-live-anita-drupal.pantheonsite.io       canonical name = fe1.edge.pantheon.io.
-Name:   fe1.edge.pantheon.io
-Address: 23.185.0.1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8001::1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8000::1
-```
+### Why does my domain say "Update Recommended?"
+
+The **Status** in **Domains / HTTPS** will show as <span class="glyphicons glyphicons-alert text-warning"></span> **Update Recommended** when the Platform detects a CNAME record pointed to Pantheon, or when A/AAAA records are not detected.
+
+Click **Details** to find the values required for A and AAAA records to add, or to find the CNAME detected on the line with `Remove this detected record` in the status.
```

Comment:
this sentence is a little awkward. did I write it? =)

Reply:
haha nah, I'll take the blame for anything that reads awkwardly - I added the `, or to find` to help separate the two things users would be looking for
Moving this to copy review for @pantheon-systems/docs-admins to help smooth and merge

Reply:
It may be a bit awkward, but it's the best kind of correct and a better phrasing doesn't come to my mind.